src/Module.Server/Common/CrpgTeamSelectComponent.cs

```diff
@@ -478,6 +481,8 @@ private byte[] GzipCompress(byte[] bytes)
 
     private class RoundResultData
     {
+        public bool LastRound { get; set; }
+        public (int defender, int attacker) RoundScore { get; set; }

```

Comment:
Can't this data be inferred from other logs? Also, this data is probably super biased, a 5-0 doesn't mean much.

If you think it's relevant, I would add a new class
```cs
class RoundResultTeamData
{
    /// <summary>Total score from the start of the mission</summary>
    public int TotalScore { get; set; }
    public List<RoundPlayerData> Players { get; set; }
}
`

Reply:
I'd like to get number of people alive at the end of the round , but i did not find a way to do it . Any idea @verdie-g

Reply:
Currently you can have that by summing people with `death == 0`. Would that work for your use-case ?

Reply:
> Currently you can have that by summing people with `death == 0`. Would that work for your use-case ?

Does this only count death in this round , or death since the start of the round?

Reply:
That should be equivalent since the data is logged at the end of the round.